builder/dockerfile/dispatchers.go

```diff
@@ -591,6 +591,7 @@ func healthcheck(req dispatchRequest) error {
 		flTimeout := req.flags.AddString("timeout", "")
 		flStartPeriod := req.flags.AddString("start-period", "")
 		flRetries := req.flags.AddString("retries", "")
+		flNoWait := req.flags.AddBool("nowait", false)
```

Comment:
Didn't realize this wasn't hyphenated. Wouldn't `no-wait` be more consistent with the other flags?